openhantek/src/utils/printutils.h

```diff
@@ -9,7 +9,7 @@
 //////////////////////////////////////////////////////////////////////////////
 /// \enum Unit utils/printutils.h
 /// \brief The various units supported by valueToString.
-enum Unit { UNIT_VOLTS, UNIT_DECIBEL, UNIT_SECONDS, UNIT_HERTZ, UNIT_SAMPLES, UNIT_COUNT };
+enum Unit { UNIT_VOLTS, UNIT_DECIBEL, UNIT_SECONDS, UNIT_HERTZ, UNIT_SAMPLES, UNIT_COUNT, UNIT_TIMES};
```

Comment:
UNIT_COUNT should always be the last item

Reply:
I'modify it.